### PR TITLE
Veyis practice app organized

### PR DESCRIPTION
**Pull request by veyisTurgut**
Fixed the bug mentioned in https://github.com/bounswe/2021SpringGroup12/issues/155.

### CODE REVIEW

**Comment by veyisTurgut** [comment]
It looks like I've changed everything, but it's not. I (Suzan Hoca while fixing unittest import) removed `src/` folder.

**Comment by veyisTurgut** [comment]
Here is the tree structure of backend:

image

**Comment by veyisTurgut** [comment]
Here is the tree structure of backend server:

image

File: practice-app/sqlfiles/create_tables.sql [hunk 1]
```diff
@@ -15,7 +15,7 @@ CREATE TABLE IF NOT EXISTS BookISBNs (
     book_id             INTEGER,
   	isbn                TEXT,
 	PRIMARY KEY(book_id,isbn),
-    FOREIGN KEY(book_id) REFERENCES Books(book_id), -- todo: on delete vsvs
+    FOREIGN KEY(book_id) REFERENCES Books(book_id) ON DELETE CASCADE,
```
Comment (Refref18) [inline practice-app/sqlfiles/create_tables.sql:18]: do we actually need the ON DELETE thing? do you delete books from the  books table?

File: practice-app/unittests/test_books.py [hunk 2]
```diff
@@ -0,0 +1,266 @@
+import unittest
```
Comment (Refref18) [inline practice-app/unittests/test_books.py:1]: Loved the tests it is so clear that while I did the unit test I took this as a reference !

**Review by Refref18 — APPROVED** [review]

**Comment by veyisTurgut** [comment]
There is something wrong going on here!
 My frontend codes are not in master branch. But they are here. Still it says that my branch is all behind the master.

**Comment by veyisTurgut** [comment]
> There is something wrong going on here!
> My frontend codes are not in master branch. But they are here. Still it says that my branch is all behind the master.

Resolved manually

**Pull request merged**